pymc3/external/emcee/sampling.py

```diff
@@ -0,0 +1,123 @@
+import pymc3 as pm
+from pymc3.backends import NDArray
+from pymc3.backends.base import MultiTrace
+from pymc3.external.emcee.backends import EnsembleNDArray
+from pymc3.external.emcee.step_methods import ExternalEnsembleStepShared, AffineInvariantEnsemble
+from pymc3.sampling import _update_start_vals
+from theano.gradient import np
+
+
+def get_random_starters(nparticles, model):
+    return {v.name: np.asarray([v.distribution.random() for i in range(nparticles)]) for v in model.vars}
+
+
+def build_start_points(nparticles, method='random', model=None, **kwargs):
+    if method == 'random':
+        return get_random_starters(nparticles, model)
+    else:
+        start, _ = pm.init_nuts(method, nparticles, model=model, **kwargs)
+        return {v: start.get_values(v) for v in start.varnames}
+
+
+def sample(draws=500, step='affine_invariant', init='random', n_init=200000, start=None,
```

Comment:
If the other sampler included with emcee is going to be implemented as well, then we should group their use within that `sample` function. What else could you call it? It's already imported with `from pymc3.external.emcee import sample`, so there is no confusion. I personally think that calling it something like `emcee_sample` doesn't add that much, no?